docs/users-guide/field-types.md

```diff
@@ -0,0 +1,98 @@
+# Field types in Metabase
+
+Understanding what field types are and how they function in Metabase will help you become more familiar with your own data and get the most out of Metabase.
+
+While data types indicate to a database how it should interpret the values in a field, __field types__ or __semantic types__ describe the _meaning_ of a field. For example, a column's data type could be ```type/text``` but the semantic type may be __Email__. Field types are just one example of metadata—information that about your data—that you can change in Metabase.
```

Comment:
```suggestion
While data types indicate to a database how it should interpret the values in a field, __field types__ or __semantic types__ describe the _meaning_ of a field. For example, a column's data type could be ```type/text``` but the semantic type may be __Email__. Field types are just one example of metadata—information about data—that Admins can change in Metabase.
```

We should link to the Data model page here so Admins can learn what other metadata they can modify. (I see the section below links to it, but the reader might not make it that far).